### PR TITLE
added ethereum key

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: src/sdk/controllers/UserAuthorization.ts [hunk 1]
```diff
@@ -100,4 +100,15 @@ export class UserAuthorization extends UserBase implements IUserAuthentication {
...
+    async saveEthereumKey(masterPassword: string, options: ICreateAccountOptions): Promise<void> {
+        const res = await options.keyFromPasswordFn(masterPassword);
+
+        const privateKey = res.privateKey;
+
+        console.log('privatekey', privateKey);
+        await this.keyManager.storeKey({
+            level: KeyManagerLevel.ETHEREUM_KEY,
+            privateKey,
+        });
```
Comment (sadiabbasi) [inline src/sdk/controllers/UserAuthorization.ts:112]: @theblockstalk here i creates the private key how i can fetch the private key to perform transactions? i am bit confused about this

File: src/cli/bootstrap/keys.ts [hunk 2]
```diff
@@ -13,10 +13,14 @@ const tonomyEosioProxyContract = TonomyEosioProxyContract.Instance;
...
     password: string,
-    salt?: Checksum256
+    salt?: Checksum256,
+    chainId?: string
 ): Promise<{ privateKey: PrivateKey; salt: Checksum256 }> {
     if (!salt) salt = Checksum256.from(randomBytes(32));
-    const hash = await argon2.hash(password, {
+    let passwordWords = password;
+
+    if (chainId) passwordWords = password + chainId;
```
Comment (theblockstalk) [inline src/cli/bootstrap/keys.ts:22]: check chainId is a string. if not fail

File: src/sdk/controllers/UserAuthorization.ts (hunk 1 above)
Comment (theblockstalk) [reply src/sdk/controllers/UserAuthorization.ts:112]: lets not use the KeyManager for this. as it is outside the Pangea functions. Lets just make a key manager in Tonomy ID

Comment (theblockstalk) [reply src/sdk/controllers/UserAuthorization.ts:112]: the key type `PrivateKey` used in our library is built for Antelope transaction signing. We need to convert this key type (which uses the underlying `curve` library I believe) into a format that we can send into the `web3` libraries for transaction signing

**Comment by sadiabbasi** [comment]
we managed this in out wallet app

**Pull request closed without merging**